src/BuildIntegration/Microsoft.NETCore.Native.Windows.props

```diff
@@ -67,7 +67,8 @@ See the LICENSE file in the project root for more information.
     <ItemGroup>
       <LinkerArg Condition="$(NativeLib) == 'Shared'" Include="/DLL" />
       <LinkerArg Include="@(NativeLibrary->'&quot;%(Identity)&quot;')" />
-      <LinkerArg Include="/NOLOGO /DEBUG /MANIFEST:NO" />
+      <LinkerArg Include="/NOLOGO /MANIFEST:NO" />
+      <LinkerArg Condition="$(DebugSymbols) == 'true' or ($(DebugType) != 'none' and $(DebugType) != '')" Include="/DEBUG" />
```

Comment:
It would be nice to avoid duplicating this long conditions:

- In the "set defaults for unspecified properties", add something like this: 

`<DebugSymbols Condition="$(DebugSymbols) == '' and $(DebugType) != 'none' and $(DebugType) != ''>true</DebugSymbols>`

And the change the rest to just check `DebugSymbols`.